src/main/java/org/wildfly/security/password/util/ModularCrypt.java

```diff
@@ -70,13 +59,7 @@ private PasswordUtil() {}
     private static final int A_APACHE_HTDIGEST          = 7;
     private static final int A_BSD_CRYPT_DES            = 8;
     private static final int A_CRYPT_DES                = 9;
-    private static final int A_DIGEST_MD2               = 10;
```

Comment:
As we discussed yesterday, we *could* continue to support custom types in the modular crypt string.  But if we do so, I think we should move to a more standard ``$id$content`` syntax instead of the ad-hoc mixture at present.  The only exceptions are the old crypt password types which are already recognizable in a standard way.